src/docs/product/integrations/data-visualization/amazon-sqs/index.mdx

```diff
@@ -10,7 +10,7 @@ description: "Learn about how Sentry's Amazon SQS integration is useful for a mo
 
 Amazon SQS is useful for a more in-depth analysis of exceptions, making it quick and easy to pipe exceptions back into your own systems. Or, use it to empower other teams, such as a Business Intelligence function.
 
-This integration needs to be set up in each project for which you wish to use it. It is maintained and supported by the [Sentry community](https://forum.sentry.io/).
+This integration needs to be set up in each project for which you wish to use it. It is maintained and supported by the Sentry community.
```

Comment:
Should we add this link here? Same thought for the next three files. 

```suggestion
This integration needs to be set up in each project for which you wish to use it. It is maintained and supported by the [Sentry community](https://open.sentry.io/).
```

Reply:
➕1 on adding the link to open.sentr.io here to replace the forum link.